site/en/docs/extensions/reference/cookies/index.md

```diff
@@ -22,6 +22,24 @@ to access. For example:
 }
 ```
 
+## Partitioning {: #partitioning }
+
+{% Aside %}
+We recently published an [API proposal][partitioning-api-proposal] for changes to the
+`chrome.cookies` API that would improve support for partitioning.
+{% endAside %}
+
```

Comment:
```suggestion
```

Reply:
https://developers.google.com/style/timeless-documentation

Reply:
We specifically discussed this wording in our meeting (changed from previous wording which said we were actively working on changes). With that context I'd like to keep this.

Reply:
Removed this.